src/components/bottomDrawer/stage/stage.js

```diff
@@ -29,9 +29,9 @@ class Stage extends Component {
               textColor="primary"
               scrollButtons="on"
               scrollable>
-          <Tab label="Friday July 12th" />
-          <Tab label="Saturday - July 13th" />
-          <Tab label="Sunday - July 14th" />
+          <Tab label="Fri July 12th" />
+          <Tab label="Sat - July 13th" />
```

Comment:
@ccabanero I think there should be some continuity on using/not using hyphens to separate the day of week (EX: Sat) with the date (EX: July 13)